YoCode/TestCountCheck.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace YoCode
+{
+    public class TestCountCheck
+    {
+        string processName;
+        string workingDir;
+        string arguments;
+
+        public string statLine { get; set; }
+        public string Output { get; set; }
+
+        TestStats stats;
+        List<int> tempStats;
+
+
+        public TestCountCheck(string repositoryPath)
+        {
+            processName = "dotnet";
+            workingDir = repositoryPath;
+            arguments = "test";
+        }
+
+        public void ExecuteTheCheck()
+        {
```

Comment:
This is also never used but I guess that's just because it hasn't been wired up yet?

Reply:
correct